src/routes/plans/+page.svelte

```diff
@@ -644,174 +657,199 @@
-                use:tooltip={{
-                  content: 'Export Selected Plan',
-                  placement: 'top',
-                }}
-              >
+          <div class="flex gap-1">
+            <div
+              use:tooltip={{
+                content: 'Export Selected Plan',
+                placement: 'top',
+              }}
+            >
+              <Button variant="outline" disabled={planExporting} class="flex gap-1" on:click={onExportSelectedPlan}>
                 <ExportIcon /> Export{#if planExporting}ing...{/if}
-              </button>
+              </Button>
+            </div>
+
+            <div use:tooltip={{ content: 'Deselect plan', placement: 'top' }}>
+              <Button size="icon" variant="ghost" disabled={planExporting} on:click={deselectPlan}>
+                <X size={12} />
+              </Button>
             </div>
-            <button
-              class="st-button icon fs-6"
-              on:click={deselectPlan}
-              use:tooltip={{ content: 'Deselect plan', placement: 'top' }}
-            >
-              <XIcon />
-            </button>
           </div>
         {:else}
           <SectionTitle>New Plan</SectionTitle>
-          <button
-            class="st-button secondary transfer-button"
-            type="button"
-            on:click={isPlanImportMode ? hideImportPlan : showImportPlan}
+          <div
+            use:permissionHandler={{
+              hasPermission: canCreate,
+              permissionError,
+            }}
           >
-            <ImportIcon /> Import
-          </button>
+            <Button
+              disabled={!canCreate}
+              type="button"
+              on:click={isPlanImportMode ? hideImportPlan : showImportPlan}
+              class="gap-1"
+              variant="outline"
+            >
+              <Import size={16} /> Import
+            </Button>
+          </div>
         {/if}
       </svelte:fragment>
 
       <svelte:fragment slot="body">
         {#if selectedPlan}
           <div class="plan-metadata">
             <fieldset>
-              <Input layout="inline">
-                <label class="plan-metadata-item-label" for="name">Model</label>
-                <input
-                  disabled
-                  class="st-input w-full"
-                  name="name"
-                  use:tooltip={{ content: selectedPlanModelName, placement: 'top' }}
-                  value={selectedPlanModelName}
-                />
-              </Input>
-              <Input layout="inline">
-                <label class="plan-metadata-item-label" for="id">Name</label>
-                <input disabled class="st-input w-full" name="id" value={selectedPlan.name} />
-              </Input>
-              <Input layout="inline">
-                <label class="plan-metadata-item-label" for="start-time">
+              <!-- TODO consider creating a Layout component to handle stacked vs horizontal label + inputs? -->
```

Comment:
Isn't this what the `Input` component already handles?

Reply:
Good point, I wonder if we should in the future rename our `Input` to be something like `InputLayout` since it isn't really an input..

Reply:
Used our internal Input component here, should be resolved.